pkg/util/containers/network.go

```diff
@@ -0,0 +1,100 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+
+package containers
+
+import (
+	"bufio"
+	"encoding/binary"
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// DefaultGateway returns the default Docker gateway.
+func DefaultGateway() (net.IP, error) {
```

Comment:
I think you might need to merge master in, this code has already merged

Reply:
i'm actually moving it from util/docker to util/containers, because otherwise we get an import cycle